app/presenters/queries/content_item_presenter.rb

```diff
@@ -13,6 +13,7 @@ def self.present(content_item)
       end
 
       def self.present_many(content_item_scope, fields: nil, order: { public_updated_at: :desc }, offset: 0, limit: nil, locale: nil)
+        content_item_scope = State.filter(content_item_scope, name: ["draft", "published"])
```

Comment:
Is it possible to put this in the GetContentCollection#content_items. Then it would work for the total too?